solidity-v1/dashboard/src/components/modal/staking/ConfirmDelegation.jsx

```diff
@@ -23,9 +38,15 @@ export const ConfirmDelegation = withBaseModal(
             that time, you can undelegate your stake.
           </p>
           <OnlyIf condition={isFromGrant}>
-            <p className="text-black mt-1 text-bold">
-              To use this stake in Threshold, you will first have to contact
-              your grant manager. Please do it immediately after you stake.
+            <p style={styles.warning}>
+              Please do not forget to contact your Grant Manager{" "}
+              <a
+                style={styles.grantManager}
+                href={`mailto:${GRANT_MANAGER_EMAIL}`}
+              >
+                ({GRANT_MANAGER_EMAIL})
```

Comment:
I think we should move the brackets outside of the `<a>` tag.

Reply:
[244a15e](https://github.com/keep-network/keep-core/pull/2853/commits/244a15ea0b9c2192fc473b26ba4b2e3520724f3a)